.github/workflows/file_verification.yml

```diff
@@ -0,0 +1,61 @@
+name: Lance File Write Read Round Trip Verification
+on: [pull_request]
+
+jobs:
+  run:
+    runs-on: "ubuntu-24.04"
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
+      - name: Install Build Requirements
+        run: |
+          sudo apt update
+          sudo apt install -y protobuf-compiler
+
+      - name: Check AWS_ACCESS_KEY_ID
+        run: |
+          if [ -z "${{ secrets.LANCEDB_S3_READER_ACCESS_KEY }}" ]; then
+            echo "AWS_ACCESS_KEY_ID is not set"
+            exit 1
+          else
+            echo "AWS_ACCESS_KEY_ID is set"
+          fi
+
+      - name: Check AWS_SECRET_ACCESS_KEY
+        run: |
+          if [ -z "${{ secrets.LANCEDB_S3_READER_SECRET }}" ]; then
+            echo "AWS_SECRET_ACCESS_KEY is not set"
+            exit 1
+          else
+            echo "AWS_SECRET_ACCESS_KEY is set"
+          fi
+
+      - name: Test Lance File
```

Comment:
I would break this out into multiple steps. It might be annoying to look at the CI output, because in order to see the test results you need to scroll past the output of `maturin develop --release`, which can be quite long. I think at the very least the build should be a separate step from the tests. But I would also considering making the file download it's own step too.

Reply:
yeah, it's better to split this up, I am not familiar with how github ci workflow works and actually didn't know the implications of making multiple steps. thanks!

Reply:
fixed